### PR TITLE
All completed and formated

### PR DESCRIPTION
**Pull request by AdriaCParra**

### CODE REVIEW

**Review by GALLEGOOOO — APPROVED** [review]
You dont have any Actions

**Review by GALLEGOOOO — CHANGES_REQUESTED** [review]
incorrect workflow path

File: exercise_07/isTypeOf.js [hunk 1]
```diff
@@ -6,3 +6,15 @@
...
  * output:
  */
+
+/**
+ *
+ * @param {*} value
+ * @param {boolean} type
+ * @returns
+ */
+function isTypeOf(value, type) {
```
Comment (tictools) [inline exercise_07/isTypeOf.js:16]: This logic does not work for array values.

FIX: needs word

File: github/.github/workflows/pipeline.yaml [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+          fetch-depth: 0
+      - name: Install node
+        uses: actions/setup-node@v3
+        with:
+          node-version: 20.9.0
+          cache: npm
+      - name: Install dependencies
+        run: npm install
+      - name: Lint code
+        run: npm run lint:fix
```
Comment (tictools) [inline github/.github/workflows/pipeline.yaml]: This is not the requested script to be executed.

FIX: define correct script

File: github/.github/workflows/pipeline.yaml
Comment (tictools) [inline github/.github/workflows/pipeline.yaml:1]: Your job is not automatically executed.

FIX: review folder structure

**Review by tictools — CHANGES_REQUESTED** [review]

File: github/.github/workflows/pipeline.yaml (hunk 2 above)
Comment (AdriaCParra) [reply github/.github/workflows/pipeline.yaml]: which is the correct one?

**Review by tictools — APPROVED** [review]

**Review by GALLEGOOOO — APPROVED** [review]

**Pull request merged**